app/views/spots/edit.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="row">
+  <div class="columns medium-8 medium-centered">
+
+  <h1 class="page-header">Edit This Spot</h1>
+
+  <% if @errors %>
+    <div class="panel callout alert">
+      <ul>
+        <% @errors.each do |error|%>
+          <li><%= error %></li>
+        <% end %>
+      </ul>
+    </div>
+  <%end%>
+
+  <%= form_for(@spot, :html => { :class => 'form' }) do |f| %>
+    <div class="field">
+      <%= f.label :name, :class => "label" %>
+      <%= f.text_field :name %>
+    </div>
+
+    <div class="field">
+      <%= f.label :location, :class => "label" %>
+      <%= f.text_field :location %>
+    </div>
+
+    <div class="field">
+      <%= f.label :photo, :class => "label" %>
+      <%= f.file_field :photo %>
+    </div>
+
+    <div class="field">
+      <%= f.label :artist, :class => "label" %>
+      <%= f.text_field :artist %>
+    </div>
+
+    <div class="field">
+      <%= f.label :description, :class => "label" %>
+      <%= f.text_field :description %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Update", :class => "button form-submit" %>
+    </div>
+  <% end %>
+
+  </div>
+</div>
```

Comment:
Please do not use `=>` (arrow) syntax, is this is very old school and deprecated. Usually you can use normal key/value pair syntax